Makefile

```diff
@@ -46,7 +54,7 @@ build: ## Build the project locally
 .PHONY: install
 install: ## Build and install the binary with the current source code. Use it to test your changes locally.
 	make build
-	cp ./bin/kubebuilder $(shell go env GOPATH)/bin/kubebuilder
+	cp ./bin/kubebuilder $(GOBIN)/kubebuilder
 
```

Comment:
This is the Makefile with the tool target and not the Makefile that is scaffolded for the projects. 
I understand that you would like to change the Makefile which is scaffolded for the operator's projects. Am I right? 
if yes, you need to apply the changes in: https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/scaffold/internal/templates/makefile.go and then, run `make generate` to update the testdata dir with mock projects.

Reply:
ah crap, there are multiple makefiles? I need to check this. Thanks @camilamacedo86 for spotting this! I will go through the Makefile throughly and try to figure out the connection and what's happening.

In the meantime, I will add a `WIP` on this one.